scripts/plot_arrows.py

```diff
@@ -0,0 +1,108 @@
+import pandas as pd
+import os
+import networkx as nx
+import numpy as np
+import matplotlib.pyplot as plt
+import matplotlib.patches as mpatches
+
+os.chdir('C:/Users/lmillischer/PycharmProjects/eutl_orm')
+
+plot1 = False
+plot2 = True
+
+# read data
+df = pd.read_csv('data/transactions_for_arrowgraph.csv', sep=';')
```

Comment:
```suggestion
from attic import paths
df = pd.read_csv(paths.path_data / 'transactions_for_arrowgraph.csv', sep=';')
```
Tu peux utiliser les paths (relatifs) définis dans `attic`. Le "/" est l'opérateur de navigation de path utilisé par pathlib (comme quand on tape cd/xyz).

Reply:
Dans l'autre fichier j'ai mis des paths relatifs